advanced/Scripts/version.sh

```diff
@@ -17,12 +17,12 @@ cachedVersions="/etc/pihole/versions"
 
 if [ -f ${cachedVersions} ]; then
     # shellcheck disable=SC1090
-    . "$cachedVersions"
+    source "$cachedVersions"
```

Comment:
Unrelated to this PR but I wonder why this is in the "$var" format and not the "${var}" format.

Reply:
https://github.com/pi-hole/pi-hole/pull/4959